### PR TITLE
Integrating API

### PR DESCRIPTION
**Pull request by Bigizi**
**Leader-Borad-Project**
I added three js files userData, getData, and showData.
I also integrated the API.

### CODE REVIEW

File: src/modules/getData.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const getData = async () => {
+  const response = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0C2VkIG8dHneddUlh9IB/scores/',
+  );
+  const endScore = await response.json();
+  return endScore.result;
+};
+
```
Comment (fobadara) [inline src/modules/getData.js:8]: [REQUIRED] Data is not displayed on the leaderboard when I submit or click refresh.

leaderb

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Bigizi, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/getData.js (hunk 1 above)
Comment (Bigizi) [reply src/modules/getData.js:8]: @fobadara When I run code on my side it is working I don't know it is not working on your side

Comment (Bigizi) [reply src/modules/getData.js:8]: Could you please run build because on my side everything is working properly @fobadara

**Review by andgarzonmal — APPROVED** [review]
Hi @Bigizi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**